study.md

```diff
@@ -147,5 +163,5 @@ using `hank` that equals "My name is Hank Hill and I'm a propane and propane
 accesories salesman that enjoys drinking beer.".
 
 ```javascript
-// your answer here
+hank = "My name is " + hank.given-name + hank.family_name + " and I'm a propane and propane accesories salesman that enjoys drinking beer."
```

Comment:
You're reassigning `hank` and also using string concatenation instead of a template literal.  `hank.given-name` will cause an error for the reasons previously stated.

```javascript
const string = `My name is ${hank['given-name']} ${hank.family_name} and I'm a ${hank.occupation} that enjoys ${hank.hobbies[0]}.`;
```